test.js

```diff
@@ -2,6 +2,10 @@ function a() {
   console.log("asd" + 'dsa');
 }
 
+function b() {
```

Comment:
![Codacy](http://localhost:9000/assets/images/favicon.png) Issue found: ["b" is defined but never used](http://localhost:9000/app/johann_8/js-tests2/file/45/issues/source?bid=1&fileBranchId=3#l5)